### PR TITLE
Expose create commands only on default folders

### LINKED ISSUE #852: Template Commands should be surfaced on only default folders.
Currently, the "SFDX: Create X" commands are shown on every folder in the solution. This is because we allow users to have a flexible structure to their project. However, this is resulting in a very overloaded right-click menu. The proposal is to change it so that the "SFDX: Create X" commands only show up on the default folder for each type. For example, "SFDX: Create Lighting Component" only shows up on the aura folder. "SFDX: Create Apex Class" only shows on the "classes" folder.

### PR DESCRIPTION
**Pull request by ntotten**
### What does this PR do?
Limits create commands to only be exposed on default folders

### What issues does this PR fix or reference?
Closes #852, @W-5746531@

### CODE REVIEW

File: packages/salesforcedx-vscode-core/package.json [hunk 1]
```diff
@@ -116,15 +116,15 @@
       "explorer/context": [
         {
           "command": "sfdx.force.apex.class.create",
-          "when": "explorerResourceIsFolder && resourceFilename != aura && resourceFilename != lwc && sfdx:project_opened"
+          "when": "explorerResourceIsFolder && resourceFilename == classes && sfdx:project_opened"
```
Comment (lcampos) [inline packages/salesforcedx-vscode-core/package.json]: This is good but looks like it breaks a test so need to get that updated as well.

Comment (brpowell) [reply packages/salesforcedx-vscode-core/package.json]: Looks good to me too. The test failure doesn't look related to these changes, I was seeing that fail locally for me on another branch as well...

File: packages/salesforcedx-vscode-core/package.json [hunk 2]
```diff
@@ -116,15 +116,15 @@
...
         },
         {
           "command": "sfdx.force.visualforce.component.create",
-          "when": "explorerResourceIsFolder && resourceFilename != aura && resourceFilename != lwc && sfdx:project_opened"
+          "when": "explorerResourceIsFolder && resourceFilename != components && sfdx:project_opened"
```
Comment (praksb) [inline packages/salesforcedx-vscode-core/package.json]: shouldn't this be resourceFilename == components

File: packages/salesforcedx-vscode-core/package.json [hunk 3]
```diff
@@ -116,15 +116,15 @@
...
         },
         {
           "command": "sfdx.force.visualforce.page.create",
-          "when": "explorerResourceIsFolder && resourceFilename != aura && resourceFilename != lwc && sfdx:project_opened"
+          "when": "explorerResourceIsFolder && resourceFilename != pages && sfdx:project_opened"
```
Comment (praksb) [inline packages/salesforcedx-vscode-core/package.json]: resourceFilename == pages

File: packages/salesforcedx-vscode-core/package.json [hunk 4]
```diff
@@ -144,7 +144,7 @@
         },
         {
           "command": "sfdx.force.apex.trigger.create",
-          "when": "explorerResourceIsFolder && resourceFilename != aura && resourceFilename != lwc && sfdx:project_opened"
+          "when": "explorerResourceIsFolder && resourceFilename != triggers && sfdx:project_opened"
```
Comment (praksb) [inline packages/salesforcedx-vscode-core/package.json]: resourceFilename == triggers

File: packages/salesforcedx-vscode-core/package.json (hunk 1 above)
Comment (ntotten) [reply packages/salesforcedx-vscode-core/package.json]: @brpowell Ah that, I was stumped on that one. I couldn't figure out why this change broke some random localization test.

File: packages/salesforcedx-vscode-core/package.json (hunk 4 above)
Comment (ntotten) [reply packages/salesforcedx-vscode-core/package.json]: Good catch. Fixed.

File: packages/salesforcedx-vscode-core/package.json (hunk 3 above)
Comment (ntotten) [reply packages/salesforcedx-vscode-core/package.json]: Good catch. Fixed

**Pull request merged**